validation/cross_validate_rust_python.py

```
@@ -54,12 +54,28 @@
+        classifier = cfd_python.CavitationRegimeClassifier(bubble_model, P_inf)
+
+        R_c_rust = bubble_model.blake_critical_radius(P_inf)
+        P_Blake_rust = classifier.blake_threshold()
+
+        print(f"  R_c = {R_c_rust*1e6:.4f} μm")
+        print(f"  P_Blake = {P_Blake_rust:.2f} Pa = {P_Blake_rust/1000:.2f} kPa")
+
+        diff_r = abs(R_c_rust - R_c_python)
+        diff_p = abs(P_Blake_rust - P_Blake_python)
+        if diff_r < 1e-10 and diff_p < 1e-5:
+            print(f"  Cross-validation: ✓ PASS")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)